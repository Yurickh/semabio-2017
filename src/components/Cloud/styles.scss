@import 'ambience/responsive.scss';

.cloud {
  position: absolute;
  bottom: 0;

  width: 100%;

  @include small-only {
    width: 240%;
  }
}
