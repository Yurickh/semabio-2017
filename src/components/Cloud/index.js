import React, { Component } from 'react'
import PropTypes from 'prop-types'

import cloud1x from './cloud@1x.png'
import cloud2x from './cloud@2x.png'

import './styles.css'

class Cloud extends Component {
  render() {
    return (
      <img className="cloud"
        srcSet={
          `${cloud1x} 1x,
           ${cloud2x} 2x`
        }
        src={cloud1x}
      />
    )
  }
}

export default Cloud
