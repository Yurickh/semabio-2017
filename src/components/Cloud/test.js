import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import Cloud from '../Cloud'

describe('Cloud', () => {
  it('should render correctly', () => {
    const button = shallow(<Cloud />)
    expect(toJSON(button)).toMatchSnapshot()
  })
})
