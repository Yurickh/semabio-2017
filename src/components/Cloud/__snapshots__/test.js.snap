// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cloud should render correctly 1`] = `
<img
  className="cloud"
  src="cloud@1x.png"
  srcset="cloud@1x.png 1x,
           cloud@2x.png 2x"
/>
`;
